chcemvediet/apps/anonymization/templatetags/chcemvediet/anonymization.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import lxml.html
+
+from poleno.utils.template import Library
+from poleno.utils.http import get_request
+from chcemvediet.apps.anonymization.anonymization import generate_user_pattern, ANONYMIZATION_STRING
+
+
+register = Library()
+
+@register.simple_tag(takes_context=True)
+def anonymize(context, inforequest, content):
+    request = context[u'request']
+    prog = generate_user_pattern(inforequest)
+    if not inforequest.anonymized_for(request.user) or not prog.pattern:
+        return content
+    return prog.sub(ANONYMIZATION_STRING, content)
+
+@register.simple_tag(takes_context=True)
+def anonymize_html(context, inforequest, html_content):
+    """
+    Anonymize user in each tag of html_content.
+    """
+    request = context[u'request']
+    prog = generate_user_pattern(inforequest)
+    if not inforequest.anonymized_for(request.user) or not prog.pattern:
+        return html_content
+    root = lxml.html.fromstring(html_content)
```

Comment:
Nepozral som detailne co vie `lxml.html`. Ale v com je vyhoda `lxml` oproti [`xml`]? Potrebujeme to `lxml` vobec? nestaci [`xml`]? Resp. pride mi divne, ze na velmi podobne veci pouzivame dve rozne kniznice.

[`xml`]: https://docs.python.org/2/library/markup.html

Reply:
V principe by si mal poradit aj `xml`, ma ale problem s takymi `.html` 
v ktorych je "bordel". Nie kazdy validny `.html` subor je aj validny `.xml`. A s tymto si `lxml` vie poradit.

Reply:
@viliambalaz Ok. Nebolo by potom rozumne pouzit `lxml` aj pri anonymizovani XML?

Reply:
Hlavne mi ide o to, ze je divne pouzivat dva rozne nastroje na velmi podobne veci. Jedine zeby naozaj jeden nastroj bol lepsi a jedno a druhy na druhe. Ak vsak jeden z nich vie zastupit druhy, pouzival by som z nich len jeden.

Reply:
@martinmacko47 Dobre, pouzijem `lxml` aj pri anonymizovani XML. Tym zjednotim funkcie `anonymize_xml` a `anonymize_html` do jednej, pricom pridam este jeden parameter `parser`.

Reply:
@viliambalaz Ano, to zneje rozumne.

Reply:
:ok: